### ISSUE TITLE
write coopth DPMI back-end

### ISSUE THREAD

**Issue by stsp** [dosemu2/dosemu2#1551]
... and port lio helpers to it.
Citing https://github.com/dosemu2/dosemu2/issues/925#issue-491640493
This will require writing the prot-mode coopthreads
back-end (somewhere in msdoshlp.c).
This will be a good start at seeing how comcom32
can be ported to 64bit.
Also this will be the first step at solving #422.

As I am now really starting to look
at that, its going to be a huge work on
infrastructure, on build system, on
packaging and whatever else.
Basically writing dosemu3 without ever
completing 2. :)

Status: coopth subsystem is now fully
modularized, so there are no more
blockings. The task of writing the DPMI
coopth back-end can finally be started...

**Comment by stsp** [comment]
Added coopthreadization of msdos.c.
Not sure how stable it is, but the only
way to find out - is to merge. :)

**Comment by stsp** [comment]
Tests failing:
https://app.travis-ci.com/github/dosemu2/dosemu2/builds/243153750
`E: Unable to locate package comcom32`
Seems like it complains to no GPG signature
for our PPA. Andrew, can you please disable
signature checks?

**Comment by andrewbird** [comment]
Seems to have been a transient failure, now running correctly with no changes needed. https://app.travis-ci.com/github/dosemu2/dosemu2/builds/243153750

**Comment by stsp** [comment]
Thanks for checking.
So lets assume this patch-set is
stable, if it survived the weekly
test-suit. :)

**Comment by stsp** [comment]
This surely already completed.
